### ISSUE TITLE
[Accessibility issue with Masthead Search]: 2 violations found via IBM Accessibility Checker

### ISSUE THREAD

**Issue by Daynssjal** [carbon-design-system/carbon-for-ibm-dotcom#8473]
### Description

Hello,
2 violations were reported regarding Masthead search component which we are using in our project: IBM Documentation.
These violations were found after scanning our page through IBM Equal Access Accessibility Checker chrome extension.
I then run the checker on the standalone dotcom component Masthead with open search, and saw the same 2 violations.

Screenshot attached.
<img width="958" alt="accessibility-violation-dotcom-masthead" src="https://user-images.githubusercontent.com/33804471/157833352-5855fed6-d74c-4d74-9cfb-6337d80e2a0d.PNG">

### Component(s) impacted

Masthead

### Browser

Chrome

### Carbon for IBM.com version

1.30.0

### Severity

Severity 1 = The design is broken in a critical way that blocks users from completing tasks or damages the brand. Affects major functionality, no workaround.

### Application/website

IBM documentation

### Package

@carbon/ibmdotcom-react, @carbon/ibmdotcom-styles

### CodeSandbox example

https://www.ibm.com/standards/carbon/react/iframe.html?id=components-masthead--search-open-onload 

### Steps to reproduce the issue (if applicable)

1. Go to Chrome webstore
2. Look for: IBM Equal Access Accessibility Checker  extension
3. Install the extension and restart browser
4. open the following link https://www.ibm.com/standards/carbon/react/iframe.html?id=components-masthead--search-open-onload  
5. open chrome devtools and choose Accessibility Assessment (next to console/networ/sources/performance)
6. scan the page
7. see the 2 violations 

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues

**Comment by jeffchew** [comment]
@IgnacioBecerra @ariellalgilmore does the work here fix what we're seeing here? https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues/8353

**Comment by kennylam** [comment]
@Daynssjal What version of the package are you using? Testing it on our latest and can't reproduce the error.

**Comment by Daynssjal** [comment]
Hi @kennylam, 1.30.0

**Comment by ingridsamakova** [comment]
@Daynssjal @kennylam Could you confirm in what version of carbon release this fix will be included, pls? thanks

**Comment by Daynssjal** [comment]
@ingridsamakova  it's already in the current release candidate as informed by carbon for dotcom in their slack channel https://github.com/carbon-design-system/carbon-for-ibm-dotcom/releases/tag/v1.33.0-rc.0
@kennylam thank you for the fix!
